api/src/pcapi/alembic/versions/20240206T105319_d118574a7888_add_is_visible_in_app.py

```diff
@@ -0,0 +1,22 @@
+"""
+Add isVisibleInApp to Venue
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# pre/post deployment: pre
+# revision identifiers, used by Alembic.
+revision = "d118574a7888"
+down_revision = "381fef70eb7a"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.add_column("venue", sa.Column("isVisibleInApp", sa.Boolean(), nullable=True, default=True))
```

Comment:
Si tu regardes le sql généré par cette migration `alembic upgrade 381fef70eb7a:d118574a7888 --sql;` tu verrais qu'alembic n'a pas ajouté de valeur par défaut : `ALTER TABLE venue ADD COLUMN "isVisibleInApp" BOOLEAN;`. 
Si tu changes en : 
```
op.add_column(
        "venue",
        sa.Column("isVisibleInApp", sa.Boolean(), nullable=False, server_default=expression.true()),
    )
```
Tu devrais avoir ce sql :
    `ALTER TABLE venue ADD COLUMN "isVisibleInApp" BOOLEAN DEFAULT true NOT NULL;` et tu ne devrais plus tomber sur [ce problème](https://squawkhq.com/docs/adding-required-field/)